### ISSUE TITLE
PNPM postinstall script fails V5.0.0

### ISSUE THREAD

**Issue by birdzai** [trivago/prettier-plugin-sort-imports#324]
**Environment**
 **Prettier version**: 3.4.2
 **node version**: 20.18.0
 **package manager**: pnpm@9
 **IDE**: VScode

**Describe the bug**

pnpm install fails with the following message:

```bash
node_modules/.pnpm/@trivago+prettier-plugin-sort-imports@5.0.0_prettier@3.4.2/node_modules/@trivago/prettier-plugin-sort-imports: 
Running postinstall script... ELIFECYCLE  Command failed.
```

**To Reproduce**

Bump plugin version in package.json to 5.0.0
Run pnpm install

**Expected behavior**

Plugin is installed

- [ ] I'm willing to fix this bug 🥇

**Comment by KilianShen** [comment]
same problem when use pnpm
![image](https://github.com/user-attachments/assets/21f6baad-730d-447f-afe6-8a3ca7725dc9)

**Comment by byara** [comment]
Yes, due to and issue after Svelte support. We are working on it. Hopefully we can find a solution.

**Comment by byara** [comment]
This issue should be fixed in [v5.2.0](https://github.com/trivago/prettier-plugin-sort-imports/releases/tag/v5.2.0). Please reopen if the issue is still happening.
